templates/ra_guac_autoscale_public_alb.template.cfn.yaml

```diff
@@ -0,0 +1,623 @@
+    Description: The base of the DN for all Guacamole configurations.
+    Type: String
+  GuacamoleVersion:
+    Default: 1.0.0
+    Description: Guacamole version tag. Defaults to 1.0.0
+    Type: String
+  GuacdVersion:
+    Default: 1.0.0
+    Description: Guacd version tag. Defaults to 1.0.0
+    Type: String
+  InstanceType:
+    AllowedValues:
+      - t2.micro
+      - t2.small
+      - t2.medium
+      - t2.large
+      - t2.xlarge
+      - t2.2xlarge
+      - t3.micro
+      - t3.small
+      - t3.medium
+      - t3.large
+      - t3.xlarge
+      - t3.2xlarge
+      - c5.large
+      - c5.xlarge
+      - c5.2xlarge
+      - c5.4xlarge
+      - m5.large
+      - m5.xlarge
+      - m5.2xlarge
+      - m5.4xlarge
+      - r4.large
+      - r4.xlarge
+    Default: t2.micro
+    Description: Amazon EC2 instance type for the Guacamole Instances
+    Type: String
+  KeyPairName:
+    Description: Public/private key pairs allow you to securely connect to your instance after it launches
+    Type: 'AWS::EC2::KeyPair::KeyName'
+  LdapDN:
+    Default: 'DC=ad,DC=example,DC=com'
+    Description: Distinguished Name (DN) of the LDAP directory
+    MinLength: '1'
+    Type: String
+  LdapServer:
+    Default: ad.example.com
+    Description: Name of LDAP server Guacamole will authenticate against
+    MinLength: '1'
+    Type: String
+  MaxCapacity:
+    Default: '2'
+    Description: The maximum number of instances for the autoscale group
+    MinLength: '1'
+    Type: String
+  MinCapacity:
+    Default: '0'
+    Description: The minimum number of instances for the autoscale group
+    MinLength: '1'
+    Type: String
+  PrivateSubnetIDs:
+    Description: List of Private Subnet IDs where the Guacamole instances will run
+    Type: 'List<AWS::EC2::Subnet::Id>'
+  PublicSubnetIDs:
+    Description: A list of Public subnet IDs to attach to the Application Load Balancer
+    Type: 'List<AWS::EC2::Subnet::Id>'
+  ScaleDownDesiredCapacity:
+    Default: '1'
+    Description: (Optional) Desired number of instances during the Scale Down Scheduled Action; ignored if ScaleDownSchedule is unset
+    Type: Number
+  ScaleDownSchedule:
+    Default: ''
+    Description: (Optional) Scheduled Action in cron-format (UTC) to scale down the number of instances; ignored if empty or ScaleUpSchedule is unset (E.g. "0 0 * * *")
+    Type: String
+  ScaleUpSchedule:
+    Default: ''
+    Description: (Optional) Scheduled Action in cron-format (UTC) to scale up to the Desired Capacity; ignored if empty or ScaleDownSchedule is unset (E.g. "0 10 * * Mon-Fri")
+    Type: String
+  SslCertificateName:
+    Description: The name (for IAM) or identifier (for ACM) of the SSL certificate to associate with the ALB -- the cert must already exist in the service
+    Type: String
+  SslCertificateService:
+    AllowedValues:
+      - ACM
+      - IAM
+    Default: ACM
+    Description: The service hosting the SSL certificate
+    Type: String
+  URL1:
```

Comment:
been a non-issue up to this point but would be nice to have an allowed pattern in here for http/s

Reply:
ok. added pattern to check for `https`